src/main/java/org/jabref/model/ai/AiProvider.java

```diff
@@ -3,10 +3,11 @@
 import java.io.Serializable;
 
 public enum AiProvider implements Serializable {
-    OPEN_AI("OpenAI", "https://openai.com/policies/privacy-policy/", "https://openai.com/policies/privacy-policy/"),
+    OPEN_AI("OpenAI", "https://api.openai.com/v1", "https://openai.com/policies/privacy-policy/"),
     MISTRAL_AI("Mistral AI", "https://mistral.ai/terms/#privacy-policy", "https://mistral.ai/terms/#privacy-policy"),
     GEMINI("Gemini", "https://huggingface.co/privacy", "https://ai.google.dev/gemini-api/terms"),
-    HUGGING_FACE("Hugging Face", "https://huggingface.co/api", "https://huggingface.co/privacy");
+    HUGGING_FACE("Hugging Face", "https://huggingface.co/api", "https://huggingface.co/privacy"),
+    CHAT4ALL("Chat4All", "http://localhost:4891/v1", "https://openai.com/policies/privacy-policy/");
```

Comment:
Replace the privacy policy for GPT4All with https://www.nomic.ai/gpt4all/legal/privacy-policy

Reply:
Got it, I have replaced that.